runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala

```diff
@@ -106,6 +107,7 @@ final class GrpcClientSettings private (
   val serviceDiscovery: SimpleServiceDiscovery,
   val defaultPort: Int,
   val resolveTimeout: FiniteDuration,
+  val connectionAttempts: Int = 5,
```

Comment:
is default needed here, doesn't that come from apply (config)?

Reply:
There's a path to create that doesn't go via any Config